api_coverage.md

```diff
@@ -1,3 +1,9 @@
+
+---
+**NOTE**: Information presented in this document is deprecated and **not updated** anymore. Please consult the official documentation to get
+information about [supported configuration types](https://www.dynatrace.com/support/help/manage/configuration-as-code/monaco/reference/supported-configuration#configuration-types-and-access-permissions).
```

Comment:
```suggestion
 **Please consult the official documentation for information about [supported configuration types](https://www.dynatrace.com/support/help/manage/configuration-as-code/monaco/reference/supported-configuration#configuration-types-and-access-permissions)**.
```

Reply:
IMO the originial wordings is not quite correct, this isn't so much a deprecated document of API coverage (we always had the table in the docs, even for v1), but a working document of which Config APIs aren't covered directly or by Settings. 

While it's not been updated for a while, the section which config APIs are simply not covered at all is correct, and will likely become relevant again, if they are not covered by Settings 2.0 schemas soon.

Reply:
It's deprecated because there is another/a newer document to look at.. but ok

Reply:
No, that's what I mean - there is no newer document that provides the information of this one.
The documentation page (which is older) lists all types that are possible to configure.
This document is wrongly named, but it actually lists what **isn't supported** explicitly. 

If we wanted to provide a clean version of that information the https://github.com/Dynatrace/dynatrace-configuration-as-code/blob/main/api_coverage.md#summarized-list-of-currently-unsupported-config-apis section would be the missing piece in the docs.

Reply:
oh, gotcha